integration-tests/gradle/pom.xml

```diff
@@ -104,6 +104,11 @@
             <artifactId>quarkus-junit5</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5-internal</artifactId>
+            <version>${project.version}</version>
```

Comment:
All these dependencies are managed, so the versions can be removed.

Reply:
This pom is generated by a script and all version are set. I will provide a fix for the `update-dependencies.sh` script to remove the version

Reply:
Ah, ok, didn't realize that.